package.json

```diff
@@ -51,6 +51,7 @@
         "crypto-js": "4.2.0",
         "dayjs": "1.11.10",
         "diff-match-patch-typescript": "1.0.8",
+        "fast-json-patch": "^3.1.1",
```

Comment:
do we really need this dependency? If yes, please do not use `^` and instead point to the latest version directly

Reply:
I pinned the dependency version. It is needed for proper type annotation of patches.